runtime/engine/src/main/java/io/aklivity/zilla/runtime/engine/internal/Info.java

```diff
@@ -131,11 +78,53 @@ public Info build()
             Info info = new Info(path);
             if (readonly)
             {
-                info.readWorkerCount();
+                try
+                {
+                    byte[] bytes = Files.readAllBytes(info.info);
+                    ByteBuffer byteBuf = ByteBuffer
+                        .wrap(bytes, Long.BYTES, Integer.BYTES)
+                        .order(nativeOrder());
+                    info.workerCount = byteBuf.getInt();
+                }
+                catch (IOException ex)
+                {
+                    System.out.printf("Error: %s is not readable\n", info.info);
+                }
             }
             else
             {
-                info.reset(workerCount);
+                info.workerCount = workerCount;
+                try
+                {
+                    Files.deleteIfExists(info.info);
+                    Files.createDirectories(info.info.getParent());
+                    Files.createFile(info.info);
+
+                    long processId = ProcessHandle.current().pid();
+                    try (SeekableByteChannel channel = Files.newByteChannel(info.info, APPEND))
```

Comment:
Please rename `info.info` field to `info.path`.

Reply:
done